grpc/proto/common.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
```

Comment:
Doesn't it make sense to have this inside `dronecore.proto`? What's the purpose of `dronecore.proto` otherwise? I see it as the entrypoint for the gRPC clients. Listing devices and their available plugins should be done there, in my opinion.

What do you guys think?

Reply:
I'd agree, I vote `dronecore.proto` for that.

Reply:
Thinking about it, I am wondering if the multi-device workflow will not be like this (I refer to "dronecore.proto" for the executable that will run the corresponding gRPC server):

* dronecore.proto starts listening for devices
* when a device registers to dronecore.proto, dronecore.proto gets its uuid and (maybe?) its list of compatible plugins (dronecore.proto could also have a way to recognize the device and know the compatible plugins, maybe, e.g. through libdronecore.so)
* dronecore.proto starts the plugins for the newly-connected device
* when a (frontend) client connects to dronecore.proto, it gets the list of devices and their corresponding plugins
* maybe a client can tell dronecore.proto to start a plugin for a device (basically saying "I do know this device and I can ensure you that this plugin will work")

My point there is that those interactions will be centralized in this dronecore.proto gRPC server, and that is the sole purpose of dronecore.proto.